.github/workflows/sdk-features-integration.yml

```diff
@@ -0,0 +1,47 @@
+name: SDK Features Integration
+
+on: # rebuild any PRs and main branch changes
+  pull_request:
+  push:
+    branches:
+      - main
+
+concurrency: # Auto-cancel existing runs in the PR when a new commit is pushed
+  group: ${{ github.head_ref }}
+  cancel-in-progress: true
+
+jobs:
+  build-docker-image:
+    # TODO: Change to main after merged
+    uses: temporalio/docker-builds/.github/workflows/docker-build-only.yml@just-build-image-workflow
+    with:
+      temporal-server-repo-ref: ${{github.event.pull_request.head.ref}}
+      docker-builds-repo-ref: just-build-image-workflow
+
+  feature-tests-ts:
+    needs: build-docker-image
+    uses: temporalio/sdk-features/.github/workflows/typescript.yaml@main
+    with:
+      version: 1.4.4
```

Comment:
is there @latest release tag that we can use? so we don't have to update for every release

Reply:
We're going to add this to the sdk-features side.

Reply:
what is this 1.4.4 version?

Reply:
The typescript SDK version